src/main/java/org/sonatype/nexus/ci/iq/IqPolicyEvaluatorUtil.groovy

```diff
@@ -92,7 +92,7 @@ class IqPolicyEvaluatorUtil
       throw e
     }
     else {
-      listener.logger.println Messages.IqPolicyEvaluation_UnableToCommunicate(e.message)
+      listener.logger.println ExceptionUtils.getStackTrace(e)
```

Comment:
My memory is a bit rusty in this area, but I would say that the idea here is to not fail builds for problems with the IQ server or the communication with the IQ server. Having an invalid app ID configured in Jenkins is not a problem with IQ or communication with IQ, so it should fail the build. In other words, I believe 4xx errors should be mapped to fail, not to unstable. But this should be solved in a different PR.

Reply:
> 4xx errors should be mapped to fail

I agree.  Exception handling needs a little enhancement and probably should be a different PR.